router/schema.go

```diff
@@ -80,5 +80,57 @@ CREATE TRIGGER notify_http_route_update
 	AFTER INSERT OR UPDATE OR DELETE ON http_routes
 	FOR EACH ROW EXECUTE PROCEDURE notify_http_route_update()`,
 	)
+	m.Add(2,
+		`ALTER TABLE http_routes ADD COLUMN path varchar(255) NOT NULL`,
```

Comment:
I can't remember, but looking at it we have no reason to use length restricted field.. I will make it use `text` instead.